### PR TITLE
HVB Inclusion 

### PR DESCRIPTION
**Pull request by prose11**

### CODE REVIEW

File: governance/polls/Updating Legal Agreements HVBank (RWA-009) - September 5, 2022.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-09-05T16:00:00
+end_date: 2022-09-08T16:00:00
+---
+# Poll:$GenericChange - Updating Legal Agreements HVBank (RWA-009) - September 5, 2022
```
Comment (patrick-j-govalpha) [inline governance/polls/Updating Legal Agreements HVBank (RWA-009) - September 5, 2022.md:18]: Remove "$" and make title more grammatically correct?
```suggestion
# Poll: Updating HVBank (RWA-009) Legal Agreements - September 5, 2022
```

File: governance/polls/Updating Legal Agreements HVBank (RWA-009) - September 5, 2022.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+title: Updating Legal Agreements HVBank (RWA-009) - September 5, 2022
```
Comment (patrick-j-govalpha) [inline governance/polls/Updating Legal Agreements HVBank (RWA-009) - September 5, 2022.md:2]: More grammatically correct?

```suggestion
title: Updating HVBank (RWA-009) Legal Agreements - September 5, 2022
```

File: governance/polls/Updating Legal Agreements HVBank (RWA-009) - September 5, 2022.md (hunk 1 above)
Comment (patrick-j-govalpha) [reply governance/polls/Updating Legal Agreements HVBank (RWA-009) - September 5, 2022.md:18]: Also change file name if you're updating the poll title 😃

**Comment by prose11** [comment]
Ty!

**Pull request merged**